### ISSUE TITLE
bug for format when sql contains $ placeholder

### ISSUE THREAD

**Issue by guoguozhenhaowan** [klahnakoski/mo-sql-parsing#132]
import mo_sql_parsing
mo_sql_parsing.parse('select $1 from table1') return {'select': {'value': 'col1'}, 'from': 'table1'}
mo_sql_parsing.format({'select': {'value': '$1'}, 'from': 'table1'}) return 'SELECT "$1" FROM table1'

**Comment by guoguozhenhaowan** [comment]
$1 is changed to string

**Comment by klahnakoski** [comment]
This is correct behaviour.  SQL strings are in single quotes, and identifiers are in double quotes.  In the case of `"$1"` the formatter is choosing to double-quote all but the simplest identifiers for safety.
